CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+## [6.3.8](https://github.com/ForestAdmin/forest-express-mongoose/compare/v6.3.7...v6.3.8) (2020-09-08)
+
+
+### Bug Fixes
+
+* :bug: correctly return documents when filtering on a field that is not returned ([#413](https://github.com/ForestAdmin/forest-express-mongoose/issues/413)) ([1190261](https://github.com/ForestAdmin/forest-express-mongoose/commit/119026181025e9ed2b84c76b3a0124f91176254a))
+
+## [6.3.7](https://github.com/ForestAdmin/forest-express-mongoose/compare/v6.3.6...v6.3.7) (2020-09-07)
+
+
+### Bug Fixes
+
+* :bug: allow to create documents with non-generated ids ([#407](https://github.com/ForestAdmin/forest-express-mongoose/issues/407)) ([fd2cffc](https://github.com/ForestAdmin/forest-express-mongoose/commit/fd2cffc81e59a83428fc412fd7f5f07ff3cc0dde))
+
```

Comment:
I don't know why you have such changes in this file.
This is not something expected.

Reply:
Do you need a rebase?

Reply:
@arnaudbesnier that's weird. I rebased manually, instead of using github to merge master into this branch. And now the diff is gone.